packages/market-service/src/api.ts

```diff
@@ -5,8 +5,8 @@ import { FiatPriceHistoryType, FindByFiatMarketType } from './fiat-market-servic
 export interface MarketService {
   baseUrl: string
   findAll: FindAllMarketType
-  findPriceHistoryByCaip19: PriceHistoryType
-  findByCaip19: FindByCaip19MarketType
+  findPriceHistoryByAssetId: PriceHistoryType
+  findByAssetId: FindByCaip19MarketType
```

Comment:
This is annoying - `FindByCaip19MarketType` was missed in the `caip` package major bump.

I think I'll do a separate PR and minor bump it so we don't major bump `caip` again.

Reply:
Agree, fixed up in https://github.com/shapeshift/lib/pull/607/files/4caedd442748e2406296dabda5d0a5eb6754d1bc..3fdcfecc42af90e464b59424c54f39c8df28a058